nginx/nginx.conf

```
@@ -85,6 +85,9 @@ http {
 		location ~ ^/api/[1-9]\d*/ {
 			proxy_pass http://relay;
 		}
+		location ~ ^/api/0/relays/ {
```

nit: https://nginx.org/en/docs/http/ngx_http_core_module.html#location
> If the longest matching prefix location has the “^~” modifier then regular expressions are not checked.

This avoids the usage of regex and makes nginx a tiny bit faster:

```suggestion
		location ^~ /api/0/relays/ {
```

---

Thanks!